compositor_api/src/types/register_input.rs

```diff
@@ -36,6 +36,8 @@ pub struct Mp4Input {
     pub url: Option<String>,
     /// Path to the MP4 file.
     pub path: Option<String>,
+    /// (**default=`false`**) If input should be played in the loop.
+    pub looped: Option<bool>,
```

Comment:
```suggestion
    pub loop: Option<bool>,
```
change also rest of the code, not only the public API

Reply:
loop is keyword in rust. I can't use it directly here.

I can use `serde(rename)` to name it `loop` in the public HTTP API (I checked - docs generate correctly).

Reply:
Using raw identifiers is also an option, but a very bad one in my opinion.

Reply:
https://doc.rust-lang.org/reference/keywords.html#strict-keywords

Reply:
PS It was exactly the reason I chose `looped` - I didn't want to use different names for this option for HTTP and Rust APIs

Reply:
In that case I would use `shouldLoop` for code and `loop` for http api. You should never trade the quality of public HTTP API in favor of internal implementation detail.